kfs-core/src/main/java/edu/arizona/kfs/fp/document/DisbursementVoucherDocument.java

```diff
@@ -0,0 +1,27 @@
+package edu.arizona.kfs.fp.document;
+
+import org.apache.log4j.Logger;
+import org.kuali.kfs.sys.businessobject.SourceAccountingLine;
+
+import edu.arizona.kfs.fp.businessobject.DisbursementVoucherSourceAccountingLineExtension;
+
+public class DisbursementVoucherDocument extends org.kuali.kfs.fp.document.DisbursementVoucherDocument {
+
+    private static final long serialVersionUID = 8820340507728738505L;

```

Comment:
Scott let me know that setting the serialVersionUID property prevents a warning and is a best practice, so let's leave it.